homeassistant/components/zha/core/channels/security.py

```diff
@@ -135,17 +124,17 @@ def cluster_command(self, tsn, command_id, args):
         """Handle commands received to this cluster."""
         if command_id == 0:
             state = args[0] & 3
-            async_dispatcher_send(
-                self._zha_device.hass, f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", state
-            )
+            self.async_send_signal(f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", state)
             self.debug("Updated alarm state: %s", state)
         elif command_id == 1:
             self.debug("Enroll requested")
             res = self._cluster.enroll_response(0, 0)
-            self._zha_device.hass.async_create_task(res)
+            asyncio.create_task(res)
```

Comment:
was this because there isn't a path back to hass now?

Reply:
yes. I'd like to decouple those as much as feasible. In the long run planning to make zigpy callbacks async, so we just could do an await here.

Reply:
👍